### DISCUSSION TITLE
openssl10|-light: rename and update to 1.1.x?

### DISCUSSION THREAD

**Discussion by tech189** [ScoopInstaller/Versions#524] [General]
The 1.x branch of OpenSSL has now increased past 1.0.x and so `openssl10` & `openssl10-light` need to be updated as checkver is still looking for 1.0.x. The current version in the manifest (1.0.2u) is no longer available to download, so no point keeping it (plus it's probably gone for security reasons). Shall we rename the manifests to `openssl1` and update?

**Comment by rashil2000** [comment]
Sure thing!
